dist/nearlib.js

```diff
@@ -153,6 +153,26 @@ class Account {
     async viewAccount (accountId) {
         return await this.nearClient.viewAccount(accountId);
     }
+
+    /**
+     * Returns full details for a given account.
+     * @param {string} accountId id of the account to look up
+     */
+    async getAccountDetails(accountId) {
+        const response = await this.nearClient.jsonRpcRequest('abci_query', [`access_key/${accountId}`, '', '0', false]);
+        const decodedResponse = this.nearClient.decodeResponseValue(response.response.value);
+        const result = {
+            authorizedApps : [],
+            transactions: []
+        };
+        Object.keys(decodedResponse).forEach(function(key) {
+            result.authorizedApps.push({
```

Comment:
We probably want to store public key for the access key here as well. In case user decides to deauthorize an app.

Reply:
Agree, I was going to do this when I work on the deauthorize flow so that it's more clear what the format should be.